evaluator/src/test/kotlin/DependencyRuleTest.kt

```diff
@@ -26,7 +26,7 @@ import org.ossreviewtoolkit.model.Package
 import org.ossreviewtoolkit.model.PackageReference
 
 class DependencyRuleTest : WordSpec() {
-    private val ruleSet = RuleSet(ortResult)
+    private val ruleSet = ruleSet(ortResult) { }
```

Comment:
How about moving `{ }` to the `ruleSet` function as a default argument for the `configure` parameter?